hms-api/src/repository/idea-repository.ts

```diff
@@ -156,6 +156,15 @@ export async function getIdea(id: Uuid): Promise<Idea> {
   throw new NotFoundError(`Idea with id: ${id} not found`);
 }
 
+export async function ideaExists(id: Uuid): Promise<boolean> {
+  try {
+    await getIdea(id);
+    return true;
+  } catch (error) {
+    return false;
+  }
+}
```

Comment:
this is a bit of an antipattern.  try/catch should be used for truly exceptional states.  In the case where the idea does not exist (as far as this bit of code is concerned) is not exceptional.  In that case you could easily modify this to not work using a thrown exception and instead just be something like:
```suggestion
export async function ideaExists(id: Uuid): Promise<boolean> {
const output = await dynamoDBClient.send(
    new GetItemCommand({
      TableName: process.env.IDEA_TABLE,
      Key: {id: {S: id}},
    }),
  );
  return !!output.Item;
}
```

Then you leave it up to the caller of this function to determine if the state is "exceptional" or not